DOCSP-48090 OmitEmpty global flag (#477)
mongodb/docs-golang: source/whats-new.txt

* :ref:`Version 1.12.1 <version-1.12.1>`
* :ref:`Version 1.12 <version-1.12>`

.. _golang-version-2.2:

What's New in 2.2
-----------------

The 2.2 {+driver-short+} release includes the following improvements
and fixes:

> - Adds the ``OmitEmpty`` global flag to globally omit empty values from
>   being serialized. For more information on setting the ``OmitEmpty`` global flag,
>   see the example in the :ref:`BSON Options <golang-bson-options>` section of the 
>   BSON guide. 

Reviewer: ```suggestion
- Adds the ``OmitEmpty`` BSON option to globally prevent empty values from
  being serialized. To learn more about setting the ``OmitEmpty`` global flag,
  see the :ref:`BSON Options <golang-bson-options>` section of the 
  BSON guide. 
```

* :ref:`Version 1.12.1 <version-1.12.1>`
* :ref:`Version 1.12 <version-1.12>`

.. _golang-version-2.2:

> What's New in 2.1

Reviewer: ```suggestion
What's New in 2.2
```

-----------------

The 2.2 {+driver-short+} release includes the following improvements
and fixes:

> - Adds a new ``OmitEmpty`` global flag to globally omit empty values from

Reviewer: ```suggestion
- Adds the ``OmitEmpty`` global flag to globally omit empty values from
```

>   marshaled BSON. This prevents the driver from writing null values for fields

Reviewer: ```suggestion
  being serialized. This prevents the driver from writing null values for fields
```

>   with empty values. For more information on setting the ``OmitEmpty`` global flag,

Reviewer: S: can omit this middle sentence

>   see the example in the :ref:`BSON Options<golang-bson-options>` section of the 

Reviewer: ```suggestion
  see the example in the :ref:`BSON Options <golang-bson-options>` section of the 
```
